packages/front-end/components/SchemaBrowser/SqlExplorerModal.tsx

```diff
@@ -376,7 +393,6 @@ export default function SqlExplorerModal({
                       <Flex align="center" gap="2">
                         <input
                           type="text"
-                          className="form-control"
                           value={tempName}
```

Comment:
question (nit): Should this still be named `tempName`?

Reply:
I think it makes sense to keep this. The tempName is the state for when a saved queries name is being edited, but not yet saved. If the user dismisses the changes we wipe the tempName, otherwise we use the tempName to update the form's `name` field, so I think it still makes sense.